graphql-kotlin-federation/build.gradle.kts

```diff
@@ -6,3 +6,22 @@ dependencies {
     api(project(path = ":graphql-kotlin-schema-generator"))
     testImplementation("org.junit.jupiter:junit-jupiter-params:$junitVersion")
 }
+
+tasks {
+    jacocoTestCoverageVerification {
+        violationRules {
+            rule {
+                limit {
+                    counter = "INSTRUCTION"
+                    value = "COVEREDRATIO"
+                    minimum = "0.96".toBigDecimal()
```

Comment:
shouldn't / couldn't we factorise those numbers as they are the same in all modules?